infrastructure/src/infra.rs

```diff
@@ -13,7 +13,7 @@ use crate::tracing::init_tracing;
 #[cfg(unix)]
 use tokio::signal::unix::{signal, SignalKind};
 
-const DEFAULT_BIND_ADDR: &str = "localhost:9010";
+const DEFAULT_BIND_ADDR: &str = "[::1]:9010";
```

Comment:
Whats the reason for changing this? I'm worried about ipv6 vs ipv4 issues.

Reply:
Already reverted :)